src/main/java/org/assertj/core/api/AbstractLocalDateTimeAssert.java

```diff
@@ -317,6 +321,30 @@ public SELF isNotIn(String... dateTimesAsString) {
     return isNotIn(convertToLocalDateTimeArray(dateTimesAsString));
   }
 
+  /**
+   * Verifies that the actual {@link LocalDateTime} is close to the current date and time on the UTC timezone,
+   * according to the given {@link TemporalUnitOffset}.
+   * You can build the offset parameter using {@link Assertions#within(long, TemporalUnit)} or {@link Assertions#byLessThan(long, TemporalUnit)}.
+   * <p>
+   * Example:
+   * <pre><code class='java'> LocalDateTime actual = LocalDateTime.now(Clock.systemUTC());
+   *
+   * // assertion will pass
+   * assertThat(actual).isCloseToUtcNow(within(1, ChronoUnit.SECONDS));
+   *
+   * // assertion will fail
+   * assertThat(actual.plusSeconds(2)).isCloseToUtcNow(within(1, ChronoUnit.SECONDS));
+   * </code></pre>
+   * @param offset The offset used for comparison
+   * @return this assertion object
+   * @throws NullPointerException if {@code offset} parameter is {@code null}.
+   * @throws AssertionError if the actual {@code LocalDateTime} is {@code null}.
+   * @throws AssertionError if the actual {@code LocalDateTime} is not close to the current time for a provided offset.
```

Comment:
better wording for `is not close to the current time for a provided offset.` would be `is not close to the current time by less than the given offset.`